share/completions/elm.fish

```diff
@@ -0,0 +1,24 @@
+set -l commands repl init reactor make install bump diff publish
+
+complete -c elm -f
```

Comment:
Personally, I would recommend putting `elm` in the variable like `set -l command elm`. However, till the command name is short and not gonna change then this solution may be pretty fine. 😄

Reply:
I wouldn't do that, this completion file will only ever work for elm.

My position on that idiom is that it's fake flexibility. You're never gonna change that one line to make it magically work for something else - the most common case is something like python/python2/python3, where you can use `complete --wraps` to reuse one completion script.

Or the tool renames itself entirely, and you just rename the file and do `s/-c elm/-c banana/` on it.

So the variable gives you one additional line and a lot of $command everywhere for no real benefit.